src/PsfAttributes.cc

```diff
@@ -302,24 +304,18 @@ double PsfAttributes::computeGaussianWidth(PsfAttributes::Method how) const {
     afwImage::MaskedImage<double> mi = afwImage::MaskedImage<double>(_psfImage);
     typedef afwImage::Exposure<double> Exposure;
     Exposure::Ptr exposure = makeExposure(mi);
-
     afwDetection::Footprint::Ptr foot = boost::make_shared<afwDetection::Footprint>(exposure->getBBox(
         afwImage::LOCAL));
 
-    GaussianCentroidControl ctrl;
     afwGeom::Point2D center(_psfImage->getX0() + _psfImage->getWidth()/2, 
                             _psfImage->getY0() + _psfImage->getHeight()/2);
-    afw::table::Schema schema = afw::table::SourceTable::makeMinimalSchema();
-    MeasureSources ms = MeasureSourcesBuilder().addAlgorithm(ctrl).build(schema);
-    PTR(afw::table::SourceTable) table = afw::table::SourceTable::make(schema);
-    PTR(afw::table::SourceRecord) source = table->makeRecord();
-    source->setFootprint(foot);
-    ms.apply(*source, *exposure, center);
-    afw::table::Centroid::MeasKey key = table->getSchema()[ctrl.name];
-    afw::table::Centroid::MeasValue centroid = source->get(key);
-    float const xCen = centroid.getX() - _psfImage->getX0();
-    float const yCen = centroid.getY() - _psfImage->getY0();
-
+    double x(center.getX());
+    double y(center.getY());
+    afw::image::Image<double> const& image(*_psfImage);
+    afwGeom::Point2D fitted_center = base::GaussianCentroidAlgorithm::fitCentroid(image, x, y);
```

Comment:
Cleaner to compress these two lines into one, and variables should be camelCase, instead of having underscores:
```
afwGeom::Point2D fittedCenter = base::GaussianCentroidAlgorithm::fitCentroid(*_psfImage, x, y);
```